in_text/text_data_release.yml

```diff
@@ -25,7 +25,7 @@ build-environment: Multiple computer systems were used to generate these data, i
 
 larger-cites:
   -
-    authors: ["Janet R. Barclay","Lauren E. Koenig Snyder","Simon N. Topp", "Margaux J. Sleckman","Alison P. Appling"]
+    authors: ["Barclay, J.R.", "Topp, S.N.", "Koenig Snyder, L.E.", "Sleckman, M.J.", "Sadler, J.M.", "Appling, A.P."]
     title: >-
       tbd  # TO BE COMPLETED ONCE MANUSCRIPT IS ACCEPTED
```

Comment:
not this comment. So we have a final link?

Reply:
No, we don't have a final link yet. We'll have to add that later.

Reply:
https://github.com/USGS-R/drb-gw-hw-model-data-release/pull/10/commits/8503275b99e6efe55c6c1d3951152b2782d00fb4